tests/drivers/pwm/pwm_to_gpio_loopback/prj.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ZTEST=y
+CONFIG_TEST_USERSPACE=y
+
+CONFIG_PWM=y
+CONFIG_GPIO=y
+CONFIG_NRF_REGTOOL_VERBOSITY=1
```

Comment:
probably keep this out of the prj.conf, could be added to board overlays if really needed :)